test/e2e/main/main_test.go

```diff
@@ -55,7 +55,8 @@ func TestOperatorUpgrade(t *testing.T) {
 		testKube.WaitForInfinispanCondition(spec.Name, spec.Namespace, ispnv1.ConditionWellFormed)
 	case tutils.OperatorUpgradeStageTo:
 		if tutils.CleanupInfinispan == "TRUE" {
-			defer testKube.DeleteInfinispan(spec, tutils.SinglePodTimeout)
+			defer testKube.CleanNamespaceAndLogOnPanic(tutils.Namespace, nil)
+			//defer testKube.DeleteInfinispan(spec, tutils.SinglePodTimeout)
```

Comment:
Thanks for the PR @durgeshanaokar. Can you remove all the commented lines as these no longer serve a purpose.